sample-input/sph-factors/assembly/compute-mgxs.py

```diff
@@ -0,0 +1,58 @@
+import copy
+import numpy as np
+import openmc.mgxs
+from infermc.materialize import differentiate_mgxs_lib
```

Comment:
Yes, I forgot that I left this reference to `differentiate_mgxs_lib` in here (which is still very much bleeding edge code at this point). I really don't want there to be any reference to InferMC in this new code, but have no way of differentiate MGXS libraries otherwise. Perhaps I should just remove the assembly SPH problem entirely - do you have a preference @samuelshaner @geogunow?